verify.md

```diff
@@ -1,47 +1,47 @@
 
 The GPG signatures for each PDF and ODT files are available here:
-- PDF (Light Theme) Main and Mirrors: [guide.pdf.asc](guide.pdf.asc)
-- ODT Main and Mirrors: [guide.odt.asc](guide.odt.asc)
+- <del>PDF (Light Theme) Main and Mirrors: [guide.pdf.asc](guide.pdf.asc)</del> (Currently unavailable)
+- <del>ODT Main and Mirrors: [guide.odt.asc](guide.odt.asc)</del> (Currently unavailable)
 
 The Minisign signatures for each PDF and ODT files are available here:
-- PDF (Light Theme) Main and Mirrors: [guide.pdf.minisig](guide.pdf.minisig)
-- ODT Main and Mirrors: [guide.odt.minisig](guide.odt.minisig)
+- <del>PDF (Light Theme) Main and Mirrors: [guide.pdf.minisig](guide.pdf.minisig)</del> (Currently unavailable)
+- <del>ODT Main and Mirrors: [guide.odt.minisig](guide.odt.minisig)</del> (Currently unavailable)
 
 ### How to check the integrity of the files using the SHA256 Checksums:
 
-Please do the following:
+First get the hash of your local file by following these steps for your OS:
 
 Windows:
 - From a command prompt, run ```certutil -hashfile filename.txt sha256```
-- Compare the result with the hash in the online checksum files. They should match.
+- Compare the obtained hash result of your local file to the online files published hash. They should match.
 
 MacOS:
 - From a terminal, run ```shasum -a 256 /full/path/to/your/file```
-- Compare the result with the hash in the online checksum files. They should match.
+- Compare the obtained hash result of your local file to the online files published hash. They should match.
 
 Linux:
 - From a terminal, run ```sha256sum /full/path/to/your/file```
-- Compare the result with the hash in the online checksum files. They should match.
+- Compare the obtained hash result of your local file to the online files published hash. They should match.
 
-All commits and releases on this repository are cryptographically signed and verified using the same GPG key.
+All commits and releases on this repository are cryptographically signed, and verified using the same GPG key.
 
-**Check for the "Verified" tags on each commit or release.**
+**Do check for the "Verified" tags on each commit or release.**
 
-### How to verify the the authenticity and integrity of the files using GPG:
+### How to verify the the authenticity, and integrity of the files using GPG:
 
-Now to verify the files with GPG signatures, you should first install gpg on your system:
+To verify the files with GPG signatures, you should first install gpg on your system:
 - Windows: Install gpg4win from <https://www.gpg4win.org/download.html>
 - MacOS: Install GPG Tools from <https://gpgtools.org/>
 - Linux: gpg should be installed by default
 
-Import the GPG key using the following command from a command prompt or terminal:
+Import the GPG key from a trusted source of the publisher using the following command from a command prompt or terminal:
```

Comment:
I don't trust any source. Thanks to key-pairs I don't have to anymore. Don't forget about key poisoning which was a huge issue not too long ago.